test

```diff
@@ -8,5 +8,21 @@ res=$(gofmt -d .)
 echo "$res"
 test -z "$res"
 
-echo "running tests"
+echo "Running tests"
 go test ./... -cover
+
+shopt -s nullglob
+echo "Checking docs"
+for doc in docs/*md
+do
+	echo "Checking $doc"
+	# split each doc into a bunch of tmpfiles then run ct on them
+	sed -n '/^```yaml fedora-coreos-config/,/^```/ p' < ${doc} | csplit - '/```yaml fedora-coreos-config/' '{*}' -z --prefix tmpconfig -q
+	for i in tmpconfig*
+	do
+		cat "$i" | tail -n +2 | head -n -1 | go run internal/main.go > /dev/null || cat -n "$i"
+		rm "$i"
```

Comment:
These files are not deleted if there's an error, should probably add a trap to delete them on exit and cancel it out after the loop.

Reply:
I think GH is having a problem. Look at my branch this PR is based on and it's fixed there.